propkatraj/propkatraj.py

```diff
@@ -2,22 +2,239 @@
 # Copyright (c) 2013-2017 David Dotson, Ricky Sexton, Armin Zjajo, Oliver Beckstein
```

Comment:
Does the copyright notice need to be changed? I know in the past you've added authors to it, but I thought it be better to ask.

Reply:
add yourself & change the year, please

Reply:
Do you have a preferred ordering? It's not alphabetical, the closest I could find was that it was ordered by number of commits, but I guess it's more "first author" <names> "corresponding author"?

Also should Shujie Fan be added too?

Reply:
You're thorough :-) – and thanks for remembering Shujie.

Let's do David, Irfan, Ricky, Armin, Shujie, Oliver.